### PR TITLE
Add Gradience 

### PR DESCRIPTION
**Pull request by 0xMRTT**
### Please confirm your submission meets all the criteria

- [ ] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [ ] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
PR should be made against the `new-pr` branch, not `master`

File: com.github.GradienceTeam.Gradience.json [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+            ]
+        },
+        {
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+                    "branch": "main"
```
Comment (hfiguiere) [inline com.github.GradienceTeam.Gradience.json:51]: NEVER branch. Use a tag or a specific commit.

File: com.github.GradienceTeam.Gradience.json [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+                }
+            ]
+        },
+        {
+            "name": "libsass",
+            "buildsystem": "simple",
```
Comment (hfiguiere) [inline com.github.GradienceTeam.Gradience.json:57]: this should work with `autotools`. see every other package that builds it.

File: com.github.GradienceTeam.Gradience.json [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+                {
+                    "type": "git",
+                    "url": "https://github.com/sass/libsass.git",
+                    "tag": "3.6.5"
+                }
+            ]
+        },
+        {
+            "name": "sassc",
+            "buildsystem": "simple",
```
Comment (hfiguiere) [inline com.github.GradienceTeam.Gradience.json:74]: same here.

File: com.github.GradienceTeam.Gradience.json [hunk 4]
```diff
@@ -0,0 +1,138 @@
+{
+    "app-id": "com.github.GradienceTeam.Gradience",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "gradience",
+    "finish-args": [
+        "--share=network",
```
Comment (hfiguiere) [inline com.github.GradienceTeam.Gradience.json:8]: do you really need network?

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: com.github.GradienceTeam.Gradience.json [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        "pypi-dependencies.json",
```
Comment (hfiguiere) [inline com.github.GradienceTeam.Gradience.json:28]: this manifest is missing.

File: com.github.GradienceTeam.Gradience.json (hunk 4 above)
Comment (0xMRTT) [reply com.github.GradienceTeam.Gradience.json]: Yes, because there is a kind of marketplace

File: com.github.GradienceTeam.Gradience.json (hunk 5 above)
Comment (0xMRTT) [reply com.github.GradienceTeam.Gradience.json]: Added

**Comment by 0xMRTT** [comment]
> PR should be made against the `new-pr` branch, not `master`

Fixed

File: com.github.GradienceTeam.Gradience.json (hunk 2 above)
Comment (LyesSaadi) [reply com.github.GradienceTeam.Gradience.json]: Somehow it didn't? It complained about missing autogen file. Do you have an example of a package which builds it, so I can see if I can change that easily?

**Comment by hfiguiere** [comment]
you need to reopen it now.

File: com.github.GradienceTeam.Gradience.json (hunk 1 above)
Comment (LyesSaadi) [reply com.github.GradienceTeam.Gradience.json]: Indeed, I tried to remove all main references, but forgot this one!

File: com.github.GradienceTeam.Gradience.json (hunk 2 above)
Comment (hfiguiere) [reply com.github.GradienceTeam.Gradience.json]: https://github.com/search?q=org%3Aflathub+sassc&type=code

Comment (LyesSaadi) [reply com.github.GradienceTeam.Gradience.json]: Thanks! Will look into that!

**Comment by 0xMRTT** [comment]
Omg, I closed it but I don't want to 😅

**Comment by 0xMRTT** [comment]
But i can't :-1:

**Comment by 0xMRTT** [comment]
image

**Comment by hfiguiere** [comment]
then you need to resubmit. Github really suck here.

**Pull request closed without merging**